### PR TITLE
add e2e test for space delimited JWT token

### PR DESCRIPTION
**Pull request by yangminzhu**
Fix: https://github.com/istio/istio/issues/13565

### CODE REVIEW

File: tests/integration/security/testdata/rbac/v1beta1-jwt.yaml.tmpl [hunk 1]
```diff
@@ -55,4 +56,11 @@ spec:
     from:
     - source:
         requestPrincipals: ["*"]
+  - to:
+    - operation:
+        paths: ["/permission"]
+        methods: ["GET"]
+    when:
+    - key: request.auth.claims[permission]
+      values: ["write"]
```
Comment (diemtvu) [inline tests/integration/security/testdata/rbac/v1beta1-jwt.yaml.tmpl]: Is this common knowledge that we treat `space` as deliminator, and convert string claims into string list?

Comment (yangminzhu) [reply tests/integration/security/testdata/rbac/v1beta1-jwt.yaml.tmpl]: Yeah I think so, see https://github.com/istio/istio/issues/13565 and also https://auth0.com/docs/tokens/guides/access-token/validate-access-token#check-permissions

Comment (diemtvu) [reply tests/integration/security/testdata/rbac/v1beta1-jwt.yaml.tmpl]: Ok. Let's remember to add this to release note and/or to the post for all authn/authz change.

Comment (yangminzhu) [reply tests/integration/security/testdata/rbac/v1beta1-jwt.yaml.tmpl]: Sure, I'll update this in the 1.4 release notes.

**Comment by yangminzhu** [comment]
@howardjohn Could you help review when you have time? thanks.

File: tests/common/jwt/jwt_token.go [hunk 2]
```diff
@@ -44,6 +44,18 @@ const (
...
 	TokenIssuer2 = "eyJhbGciOiJSUzI1NiIsImtpZCI6InRUX3c5TFJOclk3d0phbEdzVFlTdDdydXRaaTg2R3Z5YzBFS1I0Q2FRQXciLCJ0eXAiOiJKV1QifQ.eyJleHAiOjQ3MTU3ODI3ODMsImdyb3VwcyI6WyJncm91cC0yIl0sImlhdCI6MTU2MjE4Mjc4MywiaXNzIjoidGVzdC1pc3N1ZXItMkBpc3Rpby5pbyIsInN1YiI6InN1Yi0yIn0.KmOIdRoI-FfaWR6xdMWrguOypmqD5UC2VR6B1Cw8V0aBhaqpCBcn5fDvA13mSiU3sx3VFF7-CbnnZVz4j6k2RsYNnWDDJTzr4GnxIXJYJpIRBcE0Au9mcZM0cVhW9BBtKtc0twy0I5S7H7qIAEPBq6iYm8IED409K7YIuI92kLeSe--ehhs1yWmI-YbixYwwq7viD1ZIRdbP5b2ZxU6A0VHnWVXFBGSoYshSLdJS6fnlU480SXyz0oUakDpkaVkz6n9LXolggs6Cz6Gq9OIEs2zUz2DqAJ5h6HChjIQ8AydwQnRMx4qnEbdtqq2qy1h5HHfosq8pcGbiQKf7CSQKHw"
 
+	// Payload {
+	//  "exp": 4730732495,
+	//  "iat": 1577132495,
+	//  "iss": "test-issuer-2@istio.io",
+	//  "permission": "read write",
+	//  "sub": "sub-2"
+	// }
+	// Generated by: security/tools/jwt/samples/gen-jwt.py tests/common/jwt/key.pem -jwks=tests/common/jwt/jwks.json
```
Comment (howardjohn) [inline tests/common/jwt/jwt_token.go]: thanks!

**Pull request merged**